DevoxxClientMobile/src/main/java/com/devoxx/views/SessionsPresenter.java

```diff
@@ -338,6 +339,13 @@ private void updateSessionsDecoration(List<Session> sessions) {
                 colorFlag = !colorFlag;
             }
             session.setDecorated(colorFlag);
+
+            if (previousSessionWithType == null ||
+                    session.getStartDate().toLocalDate().isAfter(previousSessionWithType.getStartDate().toLocalDate()) ||
```

Comment:
`isAfter` is not supported on iOS. It can be replaced with `date.toEpochSecond() > otherDate.toEpochSecond()`

Reply:
I want to check if the session lies on the **next day** :)

Reply:
Ok, but `isAfter` is not supported on iOS, so you need to find a workaround either way.